backend/src/Service/Order/OrderService.php

```diff
@@ -258,4 +260,13 @@ public function orderUpdateCaptainOrderCost(OrderUpdateCaptainOrderCostRequest $
 
         return $response;
     }
+    
+    public function updateCaptainArrived(OrderUpdateCaptainArrivedRequest $request): ?OrderUpdateCaptainArrivedResponse
+    {
+        $order = $this->orderManager->updateCaptainArrived($request);
+
+        $response = $this->autoMapping->map(OrderEntity::class, OrderUpdateCaptainArrivedResponse::class, $order);
+        
+        return $response;
```

Comment:
_$response_ variable isn't necessary, just return the result of the automapping directly.